kf_lib_data_ingest/common/misc.py

```diff
@@ -128,3 +132,163 @@ def multisplit(string: str, delimiters: list):
     assert_safe_type(delimiters, list)
     regexPattern = '|'.join(map(re.escape, delimiters))
     return re.split(regexPattern, string)
+
+
+def to_camel_case(snake_str):
```

Comment:
According to https://en.wikipedia.org/wiki/Camel_case there are contexts where camel case means that the first letter is not capitalized. Can we call this to_upper_camel_case?

Reply:
👍